File: Python/URI 1020.py
n = int(input())

a = n // 365
n = n - a*365 #resto da divisão por 365

m = n // 30
n = n - m*30 #resto da divisão por 30, feito a divisão por 365 anteriormente

d = n

print('{} ano(s)'.format(a))
print('{} mes(es)'.format(m))
print('{} dia(s)'.format(d))

File: C/URI 1020.c
#include <stdio.h>
int main(void) {
  int N,anos,meses,dias;
  scanf("%d",&N);
  anos = N / 365;
  meses = (N % 365) / 30;
  dias = (N % 365) % 30;
  printf("%d ano(s)\n", anos);
  printf("%d mes(es)\n", meses);
  printf("%d dia(s)\n", dias);
  return 0;
}